reference/constraints/Choice.rst

```diff
@@ -279,6 +279,35 @@ you can pass the class name and the method as an array.
             }
         }
 
+Supplying the Choices from an Array Constant
+--------------------------------------------
+
+You can also directly provide a constant containing an array to the ``choices`` option in the annotation::
+
+    // src/AppBundle/Entity/Author.php
+    namespace AppBundle\Entity;
+
+    use Symfony\Component\Validator\Constraints as Assert;
+
+    class Author
+    {
+        const GENRES = ['action', 'comedy'];
+
+        /**
+         * @Assert\Choice(choices=Author::GENRES)
+         */
+        protected $genre;
+    }
+
+.. warning::
+
+  The constant in the option is used without quotes.
```

Comment:
what about both inside note instead of warning? warning can be misunderstood

Reply:
do you mean in the **same** note or two different notes ?

Reply:
do you mean both inside the **same** note ?

Reply:
the same i think